test_runner/regress/test_sharding.py

```diff
@@ -288,13 +289,19 @@ def shards_on_disk(shard_ids):
 
 def test_sharding_ingest(
     neon_env_builder: NeonEnvBuilder,
+    build_type: str,
 ):
     """
     Check behaviors related to ingest:
     - That we generate properly sized layers
     - TODO: that updates to remote_consistent_lsn are made correctly via safekeepers
     """
 
+    if build_type == "debug":
+        # The quantity of data isn't huge, but debug can be _very_ slow, and the things we're
+        # validating in this test don't benefit much from debug assertions.
+        pytest.skip("Avoid running bulkier ingest tests in debug mode")
```

Comment:
We usually use a decorator for this:
```python
@pytest.mark.skipif(
    os.getenv("BUILD_TYPE") == "debug", reason="Avoid running bulkier ingest tests in debug mode"
)
```

Reply:
Ah, I was searching around for skip.*debug to find something like that but didn't find it -- thanks.